src/main/java/net/sf/jabref/cli/ArgumentProcessor.java

```diff
@@ -208,7 +211,7 @@ private boolean exportMatches(List<ParserResult> loaded) {
                             matches);
                 } catch (Exception ex) {
                     System.err.println(Localization.lang("Could not export file") + " '" + data[1] + "': "
-                            + ex.getMessage());
+                            + ExceptionUtils.getStackTrace(ex));
```

Comment:
Did you changed this on purpose? All the other error reporting in this class seems to display the message instead of the stack trace (not sure what is better, but it should be consistent).